satellite/metainfo/metainfo.go

```diff
@@ -1158,6 +1163,33 @@ func (endpoint *Endpoint) GetObject(ctx context.Context, req *pb.ObjectGetReques
 
 	if pointer.Remote != nil {
```

Comment:
pointer.Type?

Reply:
oh ignore me here

Reply:
unfortunately no. With new metainfo object is keeping redundancy scheme and we need to put RS into inline and remote pointers (last segment). We are not sure what type of will be last segment.